some phrasing in the migration guides (#10083)
Qiskit/qiskit-terra: docs/migration_guides/qi_migration.rst

    The Qiskit Primitives are algorithmic abstractions that encapsulate the access to backends or simulators
    for an easy integration into algorithm workflows.

    The current pool of primitives includes **two** different types of primitives: Sampler and
    Estimator.

    Qiskit provides reference implementations in :class:`qiskit.primitives.Sampler` and :class:`qiskit.primitives.Estimator`. Additionally,
    :class:`qiskit.primitives.BackendSampler` and a :class:`qiskit.primitives.BackendEstimator` are
    wrappers for ``backend.run()`` that follow the primitives interface.

    Providers can implement these primitives as subclasses of :class:`~qiskit.primitives.BaseSampler` and :class:`~qiskit.primitives.BaseEstimator` respectebly.
    IBM's Qiskit Runtime (:mod:`qiskit_ibm_runtime`) and Aer (:mod:`qiskit_aer.primitives`) are examples of native implementations of primitives.

    This guide uses the following naming convention:

    - *Primitives* - Any Sampler/Estimator implementation using base classes :class:`~qiskit.primitives.BackendSampler` and a :class:`~qiskit.primitives.BackendEstimator`.

>     - *Reference Primitives* -  :class:`qiskit.primitives.Sampler` and :class:`qiskit.primitives.Estimator` are reference implementation that come with Qiskit.

Reviewer: ```suggestion
    - *Reference Primitives* -  :class:`qiskit.primitives.Sampler` and :class:`qiskit.primitives.Estimator` are reference implementations that come with Qiskit.
```

>     - *Aer Primitives* - The `Aer <https://qiskit.org/ecosystem/aer>`_ primitive implemntations: class:`qiskit_aer.primitives.Sampler` and :class:`qiskit_aer.primitives.Estimator`.

Reviewer: ```suggestion
    - *Aer Primitives* - The `Aer <https://qiskit.org/ecosystem/aer>`_ primitive implementations: class:`qiskit_aer.primitives.Sampler` and :class:`qiskit_aer.primitives.Estimator`.
```

    This guide uses the following naming standard to refer to the primitives:

    - *Primitives* - Any Sampler/Estimator implementation
    - *Reference Primitives* - The ``Sampler`` and ``Estimator`` in :mod:`qiskit.primitives` --> ``from qiskit.primitives import Sampler/Estimator``
    - *Aer Primitives* - The ``Sampler`` and ``Estimator`` in :mod:`qiskit_aer.primitives` --> ``from qiskit_aer.primitives import Sampler/Estimator``
    - *Qiskit Runtime Primitives* - The IBM's Qiskit Runtime primitive implementation. The ``Sampler`` and ``Estimator`` in :mod:`qiskit_ibm_runtime` --> ``from qiskit_ibm_runtime import Sampler/Estimator``

>     - *Backend Primitive* - Qiskit wrap for non-native backend primitives. The ``BackendSampler`` and ``BackendEstimator`` in :mod:`qiskit.primitives` --> ``from qiskit import BackendSampler/BackendEstimator``

Reviewer 2: ```suggestion
    - *Backend Primitives* - Qiskit wrappers for non-native backend primitives. The ``BackendSampler`` and ``BackendEstimator`` in :mod:`qiskit.primitives` --> ``from qiskit import BackendSampler/BackendEstimator``
```

Author: many wrappers?

Reviewer: one per primitive, right? `BackendSampler` and `BackendEstimator`
